### ISSUE TITLE
Add warning box on problematic publishers (knowntype>3)

### ISSUE THREAD

**Issue by rkzack** [ournews/extensions#60]
Hi @webcolor-in here's a new project that needs to be completed this week. Please review and let us know your estimate and if you can complete this week. Our estimate is 6 hours.

In both the Chrome and Firefox extensions (Our.News versions only), we want to add a small warning box on articles for certain publishers, as shown below in the example mockup.

![publisher-caution-mockup](https://user-images.githubusercontent.com/35204816/79255259-fa015c80-7e53-11ea-8356-08804d3f677d.png)

To determine if this box should be displayed, we added three new fields to the API response within the `publink` and `pub` API calls (which are already happening within the extensions):

`knowntype` -- if this value is >3,  then a box should be displayed
`typedesc` -- this is the wording to show within the box
`typehelp` -- this is the URL to link the `typedesc` text to

So in summary, if `knowntype` is greater than 3,  then display the box, and use the `typedesc` value as the wording to show within the box, and link that wording to the `typehelp` link.

**Comment by rkzack** [comment]
Please note, there've been many changes to the extensions since you last worked on them, so please make sure to grab the latest version from this GitHub repo before starting.

**Comment by webcolor-in** [comment]
Estimates is okay with me, yes I can complete within this week and upload changes.

**Comment by rkzack** [comment]
Ok great thank you, please proceed.

**Comment by webcolor-in** [comment]
Any URL where I can test this?

**Comment by rkzack** [comment]
https://www.theonion.com/3-dogs-2-clever-backyard-projects-4-candy-bars-you-fo-1842863339

https://our.news/api/?publink=https://www.theonion.com/3-dogs-2-clever-backyard-projects-4-candy-bars-you-fo-1842863339

**Comment by webcolor-in** [comment]
Doubt, the above link is not indexed so it shows "Newstrition" card as default selected and no warning should be shown in this card right? We just have to show in Summary card?

**Comment by webcolor-in** [comment]
![image](https://user-images.githubusercontent.com/55145799/79293355-25865480-7ef1-11ea-88d2-36d831cab1f8.png)
![image](https://user-images.githubusercontent.com/55145799/79293365-2dde8f80-7ef1-11ea-99ae-384b6ab82350.png)

**Comment by rkzack** [comment]
Correct to only show on Summary card for now. We may add to Newstrition card later.

Go ahead and index it, or here's another one:
https://politics.theonion.com/it-was-for-uh-medical-reasons-says-doctor-to-bori-1842783204

**Comment by webcolor-in** [comment]
Done, code pushed to branch - satya-apr-15

**Comment by rkzack** [comment]
This is working well, thank you!  Closing issue.
